darts/tests/models/forecasting/test_baseline_models.py

```diff
@@ -0,0 +1,418 @@
+        """Tests fit and predict for univariate and multivariate time series."""
+        (model_cls, model_kwargs), is_multivariate = config
+
+        # min train series length for global naive models
+        series = tg.linear_timeseries(length=icl)
+        if is_multivariate:
+            series.stack(series + 100)
+
+        model = model_cls(**model_kwargs)
+
+        # calling fit before predict
+        with pytest.raises(ValueError):
+            model.predict(n=10)
+
+        # calling fit with covariates
+        if isinstance(model, GlobalForecastingModel):
+            err_type = ValueError
+        else:  # for local models, covariates are not part of signature
+            err_type = TypeError
+        with pytest.raises(err_type):
+            model.fit(series=series, past_covariates=series)
+        with pytest.raises(err_type):
+            model.fit(series=series, future_covariates=series)
+
+        model.fit(series=series)
+        # calling predict with covariates
+        with pytest.raises(err_type):
+            model.predict(n=10, past_covariates=series)
+        with pytest.raises(err_type):
+            model.predict(n=10, future_covariates=series)
+
+        # single series predict works with all models
+        preds = model.predict(n=10)
+        preds_start = series.end_time() + series.freq
+        assert isinstance(preds, TimeSeries)
+        assert len(preds) == 10
+        assert preds.start_time() == preds_start
+        assert preds.components.equals(series.components)
+
+        if isinstance(model, LocalForecastingModel):
+            # no series at prediction time
+            with pytest.raises(err_type):
+                _ = model.predict(n=10, series=series)
+            # no multiple series prediction
+            with pytest.raises(err_type):
+                _ = model.predict(n=10, series=[series, series])
+        else:
+            preds = model.predict(n=10, series=series)
+            assert isinstance(preds, TimeSeries)
+            assert len(preds) == 10
+            assert preds.start_time() == preds_start
+            assert preds.components.equals(series.components)
+            preds = model.predict(n=10, series=[series, series])
+            assert isinstance(preds, list)
+            assert len(preds) == 2
+            assert all([isinstance(p, TimeSeries) for p in preds])
+            assert all([len(p) == 10 for p in preds])
+            assert all([p.start_time() == preds_start for p in preds])
+            assert all([p.components.equals(series.components) for p in preds])
+
+        # multiple series training only with global baselines
+        if isinstance(model, LocalForecastingModel):
+            with pytest.raises(ValueError):
+                model.fit(series=[series, series])
+        else:
+            model.fit(series=[series, series])
+
+    def test_naive_seasonal(self):
+        # min train series length for global naive models
```

Comment:
same comment as above, let's properly define `min_train_series_length` for these models